### PR TITLE
qml: settings/network: fix bugs

### PR DESCRIPTION
**Pull request by dvajdual**

### CODE REVIEW

File: GUI/QtQuick/qml/Settings/SettingsPopup.qml [hunk 1]
```diff
@@ -101,7 +102,11 @@ CustomPopup {
         settings_menu.init()
     }
 
-    Component.onCompleted: {
-        root.armoryServersModel = bsApp.getArmoryServers()
+    function updateServersModel()
+    {
+        if (root.armoryServersModel === null)
+        {
+            root.armoryServersModel = bsApp.getArmoryServers()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/SettingsPopup.qml]: The model should be re-created on each update (`getArmoryServers` sends API request)

File: GUI/QtQuick/qml/Settings/DeleteArmoryServer.qml [hunk 2]
```diff
@@ -88,7 +88,7 @@ ColumnLayout  {
 
             preferred: true
             function click_enter() {
-                armoryServersModel.del(server_index)
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/Settings/DeleteArmoryServer.qml:91]: Will this still delete a line from the model?

Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/DeleteArmoryServer.qml:91]: Yes

Comment (dvajdual) [reply GUI/QtQuick/qml/Settings/DeleteArmoryServer.qml:91]: the same with add

**Review by sergey-chernikov — APPROVED** [review]

**Pull request merged**